azure_functions_worker/dispatcher.py

```diff
@@ -294,16 +292,54 @@ async def _handle__worker_init_request(self, req):
                 result=protos.StatusResult(
                     status=protos.StatusResult.Success)))
 
-    async def _handle__worker_status_request(self, req):
+    async def _handle__worker_status_request(self, request):
         # Logging is not necessary in this request since the response is used
         # for host to judge scale decisions of out-of-proc languages.
         # Having log here will reduce the responsiveness of the worker.
         return protos.StreamingMessage(
-            request_id=req.request_id,
+            request_id=request.request_id,
             worker_status_response=protos.WorkerStatusResponse())
 
-    async def _handle__function_load_request(self, req):
-        func_request = req.function_load_request
+    async def _handle__functions_metadata_request(self, request):
+        logger.info(f"Received FunctionMetadataRequest: {self.request_id}")
+
+        metadata_request = request.functions_metadata_request
+        directory = metadata_request.function_app_directory
+        function_path = os.path.join(directory, SCRIPT_FILE_NAME)
+
+        if not os.path.exists(function_path):
```

Comment:
How do we differentiate in our telemetry on new programming model users vs legacy model users?